### DISCUSSION TITLE
Can't create 2 forks of a repo

### DISCUSSION THREAD

**Discussion by SollyBunny** [community/community#79137] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've created a fork of DDNet but because of other silly git issues have decided to do things differently. However, since the fork is includeed within a pull requests I don't want to delete it.

For some reason git wont let me create a second fork within in the UI
```
Create a new fork
A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project. [View existing forks.](https://github.com/ddnet/ddnet/forks)
Required fields are marked with an asterisk (*).
No available destinations to fork this repository.
[SollyBunny/ddnet-dead](https://github.com/SollyBunny/ddnet-dead)
```

According to [this](https://stackoverflow.com/questions/28119733/how-can-i-make-a-second-fork-of-a-github-project), the answer is no (apart from some jank solutions).

I tried doing it via cmdline:
```sh
git clone --depth 1 https://github.com/ddnet/ddnet
cd ddnet
git remote rename origin upstream
git remote add origin https://github.com/sollybunny/ddnet
git push -u origin master
```
But sadly another unexplainable error appears
```
remote: fatal: did not receive expected object 5013962480e9d97de7165f7b4ddc84a544bb0da9
error: remote unpack failed: index-pack failed
To https://github.com/sollybunny/ddnet
 ! [remote rejected]     main -> main (failed)
error: failed to push some refs to 'https://github.com/sollybunny/ddnet'
```
I assume this is because I hadn't actually commited anything to sollybunny/ddnet/master so it gets a bit confused, but I'm not really sure how to solve this

Firstly, and more importantly, I would like to be able to pull from upstream, and push to origin and do general githuby things

Secondly, if there is, could anyone give a reason to why you can't create 2 forks of 1 repo (for which I will now list some reasons you should be able)
* To archive dead code (:hand:)
* To do different things with a project (I know branches exist, but for some cases, it's not fun or possible)
* To make it as intuitive as possible to use github. To me and many others I'd think a fork is a seperate thing which is based of something else, even if it is exclusivly for pull requests, most forks are definatly more in the vein of being their own derivative projects, so not being able to create multiple doesn't make sense
* Me and many others, after not being able to find a solution will just download a zip and edit that before uploading it to a new repo. This is definatly the wrong way and removes the really cool UI that github provides for forks and means that more janky stuff has to be done to pull future commits to the upstream.

**Comment by SollyBunny** [comment]
I'm gonna have to again turn to janky solution! :)

to me this is a simple question, and it's not the first time almost menial tasks become absolutley horrible to deal with because of errors no one has heard before, or if they have have no idea how to solve.

**Comment by thockin** [comment]
I just ran smack into this.  It seems like a really pointless limitation to me.  I wanted to share a different repo fort some collaborative work with issues and PRs which were seperated from my personal fork repo.
